### ISSUE TITLE
Invite bot developers to help on Lili

### ISSUE THREAD

**Issue by hugolpz** [hugolpz/LiListsBot#2]
Hello @Kanasimi, @Lingua-libre is overheating and overloading its contributors these days. All is to build. I would like to invite you to join Dragon_bot developments. We have to deal with both wikipage, Qitems, deletions, Commons, Wikidata,  and we are a tiny community which means we can be really reactive for access rights. I'am admin there and de facto lead coordinator these days. Could be a good testing ground with Dragons_Bot a feature example of your framework.
Also good if you focus on developing your framework and guide me as you already do. It's very helpful already.
I have no idea if it may interest you, but if interested let me know. 🌻 

### Example
- [Need automated welcome message on talkpages](https://lingualibre.org/wiki/LinguaLibre:Chat_room#Metrics_.3E_Accounts_creations)

**Comment by kanasimi** [comment]
Yes, it seems a challenge to me. I may provide some supports. About the welcome bot, I need to write some new codes in https://github.com/kanasimi/wikibot . If you need a bot to send welcome messages, please tell me. 😃

**Comment by hugolpz** [comment]
Hello Kanasimi, nice to read that.

I'am a bit confused by your ecosystem : 
* cesjs : js framework for bot
  * wikibot : demo bot using cesjs
* wikiapi : js framework for bot. (if I understand well, the v2 of cesjs, is under active development)
  * Dragons Bot : bot using wikiapi (active development by newbie, me)
 
Then, why coding new features to wikibot ?

From my point of view it would be interesting to create a demo bot for wikiapi and Dragon bot you be that demo.

I'am confuse on how/where to code it, how to host a "monitoring bot", but yes. I definitively need a bot to monitor & send welcome messages.

**Comment by kanasimi** [comment]
The codes in [wikibot](https://github.com/kanasimi/wikibot) are bot examples using CeJS library and and wikiapi. There are many samples using `wiki.listen()`, e.g., https://github.com/kanasimi/wikibot/blob/master/routine/20201008.fix_anchor.js

I think the welcome bot just need to check [newusers](https://lingualibre.org/index.php?title=Special%3ALog&type=newusers&user=&page=&year=&month=-1&tagfilter=&hide_thanks_log=1&hide_patrol_log=1&hide_tag_log=1), no need to listen to every edits.
By the way, do you have the right to assignment bots?

**Comment by hugolpz** [comment]
I can get bot passwords yes. (is that your request ?)

**Comment by kanasimi** [comment]
The bot needs a bot flag to mass-edit pages. Bureaucrats can set it.

**Comment by hugolpz** [comment]
@contentmaschine, ping me here if you need guidance for Dragon Bot or [WikiapiJS-Eggs](https://github.com/hugolpz/WikiapiJS-Eggs).
